aptos-move/mvhashmap/src/unsync_map.rs

```diff
@@ -13,33 +14,46 @@ use std::{cell::RefCell, collections::HashMap, hash::Hash, sync::Arc};
 /// In this case only the latest recorded version is relevant, simplifying the implementation.
 /// The functionality also includes Executable caching based on the hash of ExecutableDescriptor
 /// (i.e. module hash for modules published during the latest block - not at storage version).
-pub struct UnsyncMap<K: ModulePath, V: TransactionWrite, X: Executable> {
+pub struct UnsyncMap<K: ModulePath, V: TransactionWrite, X: Executable, I: Copy> {
```

Comment:
Does `I` need to satisfy just Copy trait? Are we not having any trait requirements? I'm guessing AggregatorID struct need to satisfy functions.

Reply:
I do define:

I: Hash + Clone + Copy + Eq,

below, which are needed for the implementation

Copy is needed for the field itself, so is here.